nifi-commons/nifi-expression-language/src/main/java/org/apache/nifi/attribute/expression/language/StandardPropertyValue.java

```diff
@@ -134,9 +139,14 @@ public PropertyValue evaluateAttributeExpressions(final FlowFile flowFile, final
             return this;
         }
 
-        return new StandardPropertyValue(preparedQuery.evaluateExpressions(flowFile, additionalAttributes, decorator), serviceLookup, null);
+        VariableRegistry sysRegistry = variableRegistry != null ? variableRegistry : VariableRegistryUtils.createVariableRegistry();
```

Comment:
Is there any time this is designed to be null or is this just a fallback? If it does get created here, it won't have the variables from the properties values ingested by the FlowController.